pkg/server/router/machines.go

```diff
@@ -0,0 +1,62 @@
+package router
+
+import (
+	"context"
+	"encoding/json"
+
+	"github.com/pkg/errors"
+	"github.com/rs/zerolog/log"
+	"github.com/threefoldtech/grid3-go/deployer"
+	procedure "github.com/threefoldtech/tf-grid-cli/pkg/server/procedures"
+	"github.com/threefoldtech/tf-grid-cli/pkg/server/types"
+)
+
+func MachinesDeploy(ctx context.Context, client *deployer.TFPluginClient, data string) (interface{}, error) {
+	model := types.MachinesModel{}
+
+	if err := json.Unmarshal([]byte(data), &model); err != nil {
+		return nil, errors.Wrap(err, "failed to unmarshal machine model data")
+	}
+
+	model, err := procedure.MachinesDeploy(ctx, model, client)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to deploy model")
+	}
+
+	return model, nil
+}
+
+func MachinesDelete(ctx context.Context, client *deployer.TFPluginClient, data string) (interface{}, error) {
+	modelName := ""
+	err := json.Unmarshal([]byte(data), &modelName)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to unmarshal model name")
+	}
+
+	log.Info().Msgf("cancelilng project %s", modelName)
+	if err := procedure.MachinesDelete(ctx, modelName, client); err != nil {
+		return nil, errors.Wrapf(err, "failed to delete model %s", modelName)
+	}
+
+	return nil, nil
+}
+
+func MachinesGet(ctx context.Context, client *deployer.TFPluginClient, data string) (interface{}, error) {
+	modelName := ""
+	err := json.Unmarshal([]byte(data), &modelName)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to unmarshal model name")
+	}
+
+	log.Info().Msgf("getting project %s", modelName)
```

Comment:
Regarding comments. I don't see many. What would be good is adding at least info comments at the beginning of every function that is opened to the client. Then you can add more debug functions in between. The goal here is to be able to get more information on what is happening at server side whenever we have some issues or bugs. Debug logs are really for when we have a bug and when we need to know what is happening. We can then quickly set the logger to show debug messages too.